qemu/deps/cpuid/cpuid_dump.txt

```diff
@@ -43,7 +43,7 @@ x86         Haswell Intel Core Processor (Haswell)
 
 x86         SandyBridge Intel Xeon E312xx (Sandy Bridge)
   family 6 model 42 stepping 1 level 13 xlevel 0x8000000a vendor "GenuineIntel"
-  feature_edx (sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr apic cx8 mce pae msr tsc pse de fpu)
+  feature_edx (sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de fpu)
```

Comment:
I see, in the avocado code, the defaule value is  -M > rhel6.4, so I add sep to dumpfile. Also we can use +sep fix this issue.